api/controller/logs_api.proto

```diff
@@ -0,0 +1,35 @@
+syntax = "proto3";
+
+package percona.platform.dbaas.controller.v1beta1;
+
+option go_package = "controller;controllerv1beta1";
+
+import "controller/common.proto";
+import "controller/logs.proto";
+import "github.com/mwitkow/go-proto-validators/validator.proto";
+
+// LogsAPI provides APIs for reading logs.
+service LogsAPI {
+  // GetLogs gets all logs from db cluster.
+  rpc GetLogs(GetLogsRequest) returns (GetLogsResponse);
+}
+
+message GetLogsRequest {
+  // Kubernetes auth.
+  KubeAuth kube_auth = 1 [
+    (validator.field) = {
+      msg_exists: true
+    }
+  ];
+  // DB cluster name.
+  string cluster_name = 2 [
+    (validator.field) = {
+      string_not_empty: true
+    }
+  ];
+}
+
+message GetLogsResponse {
+  // Log represents list of logs.
+  repeated Log logs = 1;
+}
```

Comment:
Do we plan to add events there?

Reply:
What do you mean? Like the log level - error, warning ...?

Reply:
Kubernetes evens as visible at `kubectl describe pod XXX`

Reply:
The plan is to leave field container empty when log/logs field contains Kubernetes events from pod.

Reply:
That info should be in the field comment, not there then :)

Reply:
Done